src/js/components/Select/README.md

```diff
@@ -93,6 +93,14 @@ Whether when 'plain' it should receive a focus outline.
 boolean
 ```
 
+**label**
```

Comment:
why not using value ? I'm not sure why we need a separate prop for the label. in fact this seems not to be the label but the selected item in the select. value today can be a react node right ? so in theory you can pass anything to the selected item.